bin/plugin/commands/common.js

```diff
@@ -133,3 +96,52 @@ exports.getModuleData = async ( modulesDir ) => {
 			return firstModule.slug.localeCompare( secondModule.slug );
 		} );
 };
+
+/**
+ * Returns the list of module data for all modules.
+ *
+ * @param {string} moduleFile Modules file.
+ *
+ * @return {[]WPModuleData} Module data list.
+ */
+exports.getModuleDataFromHeaders = ( moduleFile ) => {
```

Comment:
See my comment in https://github.com/WordPress/performance/pull/662#discussion_r1126861733, you can add another function to get the full module header as a string. Then this function here can be changed to receive not the entire `moduleFile`, but rather just the `moduleHeader`.

Reply:
Addressed in https://github.com/WordPress/performance/pull/662/commits/0032104c7595910d2d52b6bfa57eeb5d633256ea